vlasiator.cpp

```diff
@@ -1187,7 +1194,15 @@ int main(int argn,char* args[]) {
       }
       vspaceTimer.stop(computedCells, "Cells");
       addTimedBarrier("barrier-after-acceleration");
-      
+     
+      phiprof::start("Diffusion");
```

Comment:
Change this to use the new timer formalism (and could put the timer object creation inside the if-clause so if diffusion is off, the object isn't even created)